src/ui/buttons/buttonTextIcone/props.tsx

```diff
@@ -0,0 +1,23 @@
+import { ReactNode } from "react";
+
+export interface ButtonTextIconeProps {
```

Comment:
rather than writing all theses props u can extend chakra props to avoid redundency and have cleaner code. @youssef-night will explain it to u